apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/components/server/BuyFundsSection.tsx

```diff
@@ -0,0 +1,61 @@
+import { ExternalLinkIcon } from "lucide-react";
+import Link from "next/link";
+import type { ChainMetadata } from "thirdweb/chains";
+import { ChainIcon } from "../../../../components/server/chain-icon";
+import { PayModalButton } from "../client/PayModal";
+import { CreditCardIcon } from "../icons/CreditCardIcon";
+import { SectionTitle } from "./SectionTitle";
+
+export function BuyFundsSection(props: { chain: ChainMetadata }) {
+  const sanitizedChainName = props.chain.name.replace("Mainnet", "").trim();
+
+  return (
+    <section>
+      <SectionTitle title="Buy Funds" />
+      <div className="border roudned-lg border-border px-4 py-10 rounded-lg flex justify-center">
+        <div className="max-w-[520px] flex flex-col items-center ">
+          <div className="flex items-center">
+            <ChainIcon
+              className="size-12 -mr-2 rounded-full border p-1"
+              iconUrl={props.chain.icon?.url}
+            />
+            <CreditCardIcon
+              bg="hsl(var(--background))"
+              className="size-12 -ml-2"
+            />
+          </div>
+
+          <div className="h-6" />
```

Comment:
I see you hate `gap` s

Reply:
yep, especially when we need different gaps between children